src/tests/mockData/reviewBookingData.js

```diff
@@ -0,0 +1,21 @@
+export default [
+  {
+    rating: '1',
+    feedback: 'nice hotel'
+  },
+  {
+    rating: 'm',
+    feedback: 'nice home',
+  },
+  {
+    rating: '12',
+    feedback: 'nice home',
+  },
+  {
+    rating: '3',
+  },
+  {
+    rating: 'm',
```

Comment:
Is this a valid rating??

Reply:
it is just testing if we give m and rating what would happen

Reply:
what happened??

Reply:
eventually, It can't pass bro. You'll get a 400 wrong request error